### DISCUSSION TITLE
Is there an x86_64 version?

### DISCUSSION THREAD

**Discussion by ian-battaglia** [regolith-linux/regolith-desktop#671] [Q&A] [answered]
Hi, apologies for opening a new discussion on what seems like a quick question. 

I'm hoping to get back into learning about development, and wanted to use Regolith as an environment to do so. I'm a Windows user, so I dualbooted my Dell XPS 15 9500, adding a new Ubuntu 22.04 install alongside Windows 11. I see on the Regolith homepage the codes for ARM and AMD architecture, but I'm running an Intel x86_64 CPU. 

Is there a release that would work on my setup? Replacing `amd64` with `x86_64` didn't work, though maybe it was silly to think it would. I had seen [this previous issue](https://github.com/regolith-linux/regolith-desktop/issues/650) about adjusting the architecture on install, and wondered if it was possible. 

Thanks,
 - I

**Comment by kgilmer** [comment, marked as answer]
Hi @MoreThanLuck , `amd64` and `x86_64` are equivalent.  Your system should work fine for learning development with Regolith :bullettrain_front: 

https://askubuntu.com/questions/601553/what-is-the-difference-between-x86-64-amd64-and-64-bit

**Reply by ian-battaglia** [reply]
Appreciate it, @kgilmer, and sorry for the silly question! 

I installed successfully, but rebooting isn't launching me into Regolith. I didn't receive any errors on install, but running `sudo apt install regolith-desktop` again tells me `regolith-desktop is already the newest version (4.2.0-1regolith).` Regolith and Regolith settings show up under Ubuntu's "Show Applications" page, however. Is there something else I should try to complete the install?

**Comment by kgilmer** [comment]
Not a silly question at all :).  The project is _for_ those less familiar and looking to learn.  I think what's going on for you is just that you need to select the Regolith session from your login screen.  This isn't documented in the new site, but you can see the screenshot in the old site that may give you the hint you need: https://regolith-linux.org/docs/getting-started/install/

**Reply by ian-battaglia** [reply]
Ah, perfect! For some reason I was expecting it to just swap over from the install alone. Thanks again!
